### ISSUE TITLE
No ability to see the same named element in a given Endevor system 

### ISSUE THREAD

**Issue by JohnBMCR** [eclipse-che4z/che-che4z-explorer-for-endevor#229]
The product provides views based on folder structures, i.e. SYSTEM/SUB-SYSTEM/TYPE . 

This though is not the way in which most 3270 ISPF/TSO users use Endevor - or at least I don't !

Most users tend to list members of a given name with a system or subsystem so that can see all the elements of that name or subset of the an elements name irrespective of TYPE's .
In addition they also require views of members the up thru' the lifecycle (i.e. A->B->D->F - > P or say C->D->F->P). 

In effect the product needs to deliver the equivalent of the 'ELEMENT SELECTION LIST' and which supports all the available search arguments. 

The views based simply on a folder structure mapping is too restrictive and presents major challenges in enticing existing ISPF/TSO users across to the product.

**Comment by nickImbirev** [comment]
Hey @JohnBMCR !

Thank you so much for creating this issue!

Right now, we are about to start working on the issue, which will allow users to see all the elements up the Endevor map properly sorted, according to the elements route in the map. I hope this make the extension more convenient for the ISPF/TSO users.

Also, VSCode itself provides huge abilities to filter the information in the tree views, for example:
<img width="374" alt="Screenshot 2022-03-18 at 11 27 41" src="https://user-images.githubusercontent.com/74672982/158986637-12971969-d51c-40e0-8d9e-d1f3b9856ba0.png">
<img width="356" alt="Screenshot 2022-03-18 at 11 27 50" src="https://user-images.githubusercontent.com/74672982/158986640-39069868-f3cc-46bc-94b1-6426de8fc142.png">

So, basically, you can just search for the elements and then enter the name of the element that you need to work with, VSCode will highlight only required folders for you.

I hope this will help you with the searching and filtering for the required information, in case you have big numbers of elements fetched. Also, I can propose you to specify more detailed search locations to have only the information, that you want to see.

At the same time, can I ask you to elaborate, which restrictions exactly did you mean, when you talked about the folder structured view?

Hope that helps

Best regards, Nikolai

**Comment by nickImbirev** [comment]
might be connected to #113
